example/macos/Podfile

```diff
@@ -0,0 +1,82 @@
+platform :osx, '10.11'
+
```

Comment:
will this pick up the common files under the `ios` folder?

Reply:
I think this file is a leftover from my macOS tests. I will remove it since we are not compatible.